src/picturepark-sdk-v1-angular/src/app/content-picker/content-picker.component.ts

```diff
@@ -31,7 +31,7 @@ import { ContentDetailDialogOptions } from 'projects/picturepark-sdk-v1-angular-
   styleUrls: ['./content-picker.component.scss'],
 })
 export class ContentPickerComponent implements OnInit, OnDestroy {
-  public basketItemsCount = 0;
+  public basketItemsCount = '0';
```

Comment:
I'm not very comfortable having a counter as a string.
I know this way we got rid of the toString call in the content-picker.component.html, but in the end, a toString call was added anyway below at line 98.